main.go

```diff
@@ -19,51 +20,38 @@ const (
 )
 
 var (
-	username = kingpin.Flag("username", "Username").String()
-	password = kingpin.Flag("password", "Password").String()
-	endpoint = kingpin.Flag("url", "URL to proxy service to, which includes the port.").Short('u').String()
-	port     = kingpin.Flag("port", "Port to expose the service to the host").Default("8080").Short('p').Int()
+	PROXY_APP_ENDPOINT string
```

Comment:
Should be camel case. These could also move into `main()`.

Reply:
Done.